grammar.pegjs

```diff
@@ -77,7 +77,7 @@ builtinCommandName
  = '[[' / '['
 
 argument "command argument"
- = commandName
+ = concatenation
```

Comment:
I believe keywords can be arguments. Not at my machine, but I think `echo done` has no issues. Let me know if I'm misunderstanding you though.